service/src/main/resources/schema/development/schema.json

```diff
@@ -3,7 +3,7 @@
   "$id": "https://app.terra.bio/schemas/datacatalog.development.json",
   "title": "catalog dataset",
   "type": "object",
-  "required": ["dct:title", "dct:description", "dct:creator", "dct:issued", "TerraDCAT_ap:hasDataCollection", "dcat:accessURL", "prov:wasGeneratedBy", "storage", "counts", "contributors"],
```

Comment:
I stole this change from @tcjiang9 's PR, because it is necessary to create Nate's datasets.